### PR TITLE
Add base class for all test classes using WatchableApp.

### PR DESCRIPTION
**Pull request by tmat**

### CODE REVIEW

File: src/Tests/dotnet-watch.Tests/Watch/Utilities/DotNetWatchTestBase.cs [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+using System;
+using System.Diagnostics;
+using System.IO;
+using Microsoft.NET.TestFramework;
+using Xunit.Abstractions;
+
+namespace Microsoft.DotNet.Watcher.Tests;
+
+public abstract class DotNetWatchTestBase : IDisposable
```
Comment (arkalyanms) [inline src/Tests/dotnet-watch.Tests/Watch/Utilities/DotNetWatchTestBase.cs:13]: nit: documentation that can be inherited as well would be useful

**Review by arunchndr — APPROVED** [review]

**Pull request merged**